internal/configurator/testdata/curated-experience-test.yaml

```diff
@@ -0,0 +1,19 @@
+kubernetes:
+  jobs:
+    - job_name_prefix: kubernetes-job
+      target_discovery:
+        pod: true
+        endpoints: true
+  curated_experience:
+    enabled: true
+    app_values:
+      - test1
+      - test2
+    source_labels:
+      - testLabel1
```

Comment:
nit: here the labels could not contain capital letters but the values could, is inverted

Reply:
fixed